packages/components/src/DatePicker/_docs/DatePicker.stories.tsx

```diff
@@ -249,3 +250,118 @@ export const Validation: Story = {
 export const DisabledDays: Story = {
   parameters: { controls: { include: /^disabled/ } },
 }
+
+export const LimitedWindowWidth: Story = {
```

Comment:
We actually have a `Component.spec.stories.tsx` naming convention for any stories that are not used in docs.
Perhaps to review later whether we want to keep the pattern, but that's what we currently have if you want to take a look :)

Reply:
Makes sense to keep it in the same convention, at least for now. Will update that :)